src/main/java/org/zalando/nakadi/cache/CacheChange.java

```diff
@@ -0,0 +1,21 @@
+package org.zalando.nakadi.cache;
+
+import java.util.List;
+
+public class CacheChange {
+    private final List<String> modified;
+    private final List<String> deletedKeys;
+
+    public CacheChange(final List<String> modified, final List<String> deletedKeys) {
```

Comment:
Does `modified` parameter also contain keys? If yes - it probably makes sense to name it to `modifiedKeys` to keep naming consistent with `deletedKeys`